frontend/src/components/namespace/List.tsx

```diff
@@ -86,12 +87,15 @@ export default function NamespacesList() {
     }, [allowedNamespaces]);
 
   return (
-    <ResourceListView
-      title={t('Namespaces')}
-      headerProps={{
-        noNamespaceFilter: true,
-      }}
-      {...resourceTableProps}
-    />
+    <>
```

Comment:
This is not needed AFAIU.

Reply:
The headerProps for titleSideActions is what lets the button render neatly next to the page title

Reply:
I think he meant the Fragment `<></>` that wraps ResourceListView

Reply:
removed the `<></>`